### PR TITLE
feat(Release): added new field 'comment' in release summary

### PR DESCRIPTION
**Pull request by akapti**
**Summary:**
Added new field "**comment**" (text-area) in release summary [Component -> Release Overview -> Summary]
Added rest API  (GET, POST and PATCH) support and documentation for the field.

**Steps to test:**
- Create new Component or Edit existing Component and then go to "Release Overview".
- Create a new Release or Open the existing release (is present).
- User should be able to see and edit the new field "`comment`" in "`Release Summary`" tab.
- User should be able to add/update the field value while creating/updating the release via rest end point.

closes : #571 

Signed-off-by: Abdul Kapti <abdul.mannankapti@siemens.com>

### CODE REVIEW

File: frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/summaryRelease.jspf [hunk 1]
```diff
@@ -12,6 +12,7 @@
 <%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
 
 <core_rt:if test="${release != null}">
+<div class="up_Summary" id="up_Summary"><p><sw360:out value="${release.comment}" maxChar="500"/></p></div>
```
Comment (maxhbr) [inline frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/summaryRelease.jspf]: Formatting:
```suggestion
    <div class="up_Summary" id="up_Summary"><p><sw360:out value="${release.comment}" maxChar="500"/></p></div>
```

And why is it limited to 500 chars ?

Comment (akapti) [reply frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/summaryRelease.jspf]: @maxhbr The 500 chars limit is present in Component description as well. (file: `/sw360-portlet/src/main/webapp/html/components/includes/components/summary.jspf`).
So to make the release description ( view mode) in sync with component, I added that char limit.
Let me know if this has to be removed.

**Comment by mcjaeger** [comment]
discussed to have a more general structure for comments (which is then also suitable for all documents, such as projects, vendors, components etc.) in some general place and then reference the structure:

Screenshot 2019-09-04 at 10 21 27

**Comment by mcjaeger** [comment]
proposed change is so different, makes sense to open another PR

**Pull request closed without merging**